### ISSUE TITLE
Copy for Open Editor Outreach

### ISSUE THREAD

**Issue by rchampieux** [ryerbanta/radian#3]
# Brainstorm of Ideas and Goals to address:
@ryerbanta and @daniellecrobinson please add to this.

![](https://media.giphy.com/media/qhDfFlWjtuipG/giphy.gif)

* Currently data practitioners are siloed and have no central way to share and discover data solutions, approaches and ideas. 
* Additionally, their informal contributions on the open web are under-discoverable and under-valued (note:  "informal" is not universally understood term).
* We aim to create a central hub for data practitioners and learners to share and discover good stuff.
* We aim to Increase attribution and recognition for informal contributions.
* Invite data practioners to share content and resources that are useful to their work and professional development. 
* Data practioners regularly discover, create and curate content on the open web.
* By telling us about it, we can make it more discoverable across communities. 
* We seek editors-at-large to nominate and curate the content they are using and creating.
* We want to leverage the discovery and curation work they are already doing.
* Need to make it sound easy and not like extra work, emphasize the payoff.
* Should sound friendly and exciting, but overly academic.

**Comment by ryerbanta** [comment]
Chipping away with some Monday morning contributions.

I'm thinking of some thematic headings for these:

# (The Call to Action) Seeking editors-at-large to nominate and curate great data-related content

## The Problem
- Data solutions, approaches and ideas are siloed, like data practitioners in their fields, and are hard to find and apply to different contexts.
- Additionally, informal contributions on the open web are under-discoverable and under-valued (note: "informal" is not universally understood term).

## Our Solution / Our Goal
- Make great data solutions and discussions easy to find by creating a central hub 
- Increase attribution and recognition for informal contributions.
- Foster a cross-silo data practitioner community built on information sharing

## Join Us / What You Can Do / Your Help / Where You Fit In 
- Tell us about the great stuff you make and use in your work (nominate)
- Add some extra info to make the great stuff understandable and easy to use for others (curate)

**Comment by rchampieux** [comment]
Thanks @ryerbanta.  I like of breaking it up into headings, remixed / narrative version below.

@daniellecrobinson @saramannheimer @swyoung please review and comment as well.

### Contribute to a community effort to increase the discovery, recognition, and reuse of great data-related content!

### The Problem

New data solutions, ideas, and approaches are hard to find.  Experts, practitioners, and learners are sharing great teaching materials, research, and applications on the open web, but these resources are often undiscoverable and undervalued across disciplines and communities of practice.  

### Introducing Radian

Radian provides a community curated hub for sharing and discovering ideas and resources about data science, visualization, management, policy, and applications for the public good.  By highlighting and connecting the work you are using and creating in your practice, learning, and advocacy we want to increase the discovery and recognition of data related contributions on the open web, and foster a cross-disciplinary community built on information sharing.  

### Contribute

Use the PressForward bookmarklet to tell us about the awesome stuff you make and use in your work, and add some extra information to make is easier to discover and use by others.  Our contributors will be recognized as Radian Editors-at-Large on our website and credited for the posts they contribute.  Go here to learn more and get started (this would obviously point to more detailed info).

In addition to specific content nominations, Radian uses the [PressForward Plugin](http://pressforward.org/the-pressforward-plugin/) to aggregate and curate content from RSS feeds.  If you have a feed, or know of a site Radian should follow, please submit the feed or site URL here.  (again, will point somewhere).

### Team

Radian is a led by Ryer Banta, Robin Champieux, Sara Mannheimer, Danielle Robinson, and Scott Young.  We are generously funded by (add preferred wording for Force and Sloan/Press Forward)

### Connect

Email, twitter, website, and Github icons/links would go here.

**Comment by rchampieux** [comment]
Some areas specific areas where I think the doc needs the teams help:

* Are "The Problem" and "Introducing Radian" the right heads for those sections? Should they be combined under "Introducing Radian".
* We're trying to get at, but don't specifically state that we're building Radian to help people work better.  Do we need to address this more specifically, I'm thinking back to our conversations with the board.
* Because readers will ask, I think we need to describe what falls under the umbrella of "data solutions, approaches, and ideas", so I pulled used our original themes to flesh this out.  Does this work, is it clear, or does it sound too restricting?

**Comment by ryerbanta** [comment]
Great edits @rchampieux I like these headings a lot and I vote for keeping "The Problem" and "Introducing Radian" separate. That will keep it more focused and readable. Plus, both of these sections may benefit from just a little more information. 

I think this is getting really close, but we frequently struggle with the problem of this big, squishy, inclusive idea that we understand the shape of, but we can't be sure that our audience will get it quite so quickly. I think that the call to action (the first big heading) is pretty spot-on and will hopefully resonate with our target audience. Perhaps what it needs is re-iterating that language in spots.

It makes sense to me to use the language of the themes in the description of Radian.

I'll share some edits/ideas shortly today. One other thing, I think the order of team should not be alphabetical. This is Robin and Sara's main thing and I think you two should get top billing.

**Comment by ryerbanta** [comment]
Here are some options. Nit picky wordsmithing - is it better to talk about data-related content or data-related resources? To me, content seems a bit more passive, whereas resources are things you can do stuff with, make use of. In this version I'm trying to use resources and keep using the same terminology so that everyone knows we're still talking about the same thing.


# Contribute to a community effort to increase the discovery, recognition, and reuse of great data-related resources!

## The Problem

Great data-related resources are out there. Many you, data experts, practitioners, and learners alike, already share your data solutions, approaches and ideas on the open web. However, finding these great data-related resources is hard, especially if they come from another field. Plus the effort you put into creating and sharing your data resources may be undervalued across disciplines and communities of practice.

## Introducing Radian

Radian provides a community curated hub for sharing resources and discovering ideas about data science, visualization, data management, data policy, and applications for the public good. By highlighting and connecting the work you are using and creating in your practice, learning, and advocacy we want to increase the discovery and recognition of data-related contributions on the open web, and foster a cross-disciplinary community built on information sharing.

## Contribute

Use the PressForward bookmarklet to tell us about the awesome stuff you make and use in your work, and add some extra information to make is easier to discover and use by others. Our contributors will be recognized as Radian Editors-at-Large on our website and credited for the posts they contribute. Go here to learn more and get started (this would obviously point to more detailed info).

In addition to specific resource nominations, Radian uses the PressForward Plugin to aggregate and curate content from RSS feeds. If you have a feed, or know of a site Radian should follow, please submit the feed or site URL here. (again, will point somewhere).

## Team

Radian is a led by Robin Champieux and Sara Mannheimer, with additional help from Ryer Banta, Danielle Robinson, and Scott Young. We are generously funded by (add preferred wording for Force and Sloan/Press Forward)

## Connect

Email, twitter, website, and Github icons/links would go here.

**Comment by daniellecrobinson** [comment]
Hello! I have made some edits and fixed some typos - what do you think @rchampieux @ryerbanta 
# Contribute to a community effort to increase the discovery, recognition, and reuse of great data-related resources!

## The Problem

Great data-related resources are out there. Many of you - data experts, practitioners, and learners alike -  already share your data solutions, approaches, and ideas on the open web. However, finding these resources is hard, especially in cases where a resource with potential for wide appeal doesn't get shared to the wider data community.  Siloing of data related resources means that the effort you put into creating and sharing data resources in your community may be duplicated across disciplines and communities of practice. We want to break down disciplinary silos and create a space to share resources, amplify your work, and build interdisciplinary community.

## Introducing Radian

Radian provides a community curated hub for sharing resources and discovering ideas about data science, visualization, data management, data policy, and applications for the public good. By highlighting and connecting the work you are using and creating in your practice we want to increase the discovery and recognition of data-related contributions on the open web, and foster a cross-disciplinary community built on information sharing.

## Contribute

Are you ready to amplify a great data-related resource? Use the PressForward bookmarklet to tell us about the awesome stuff you make and use in your work, and add some extra information to make is easier to discover and use by others. Our contributors will be recognized as Radian Editors-at-Large on our website and credited for the posts they contribute. Go here to learn more and get started (this would obviously point to more detailed info).

In addition to specific resource nominations, Radian uses the PressForward Plugin to aggregate and curate content from RSS feeds. If you have a feed, or know of a site Radian should follow, please submit the feed or site URL here. (again, will point somewhere).

## Team

Radian is a led by Robin Champieux and Sara Mannheimer, with additional help from Ryer Banta, Danielle Robinson, and Scott Young. We are generously funded by (add preferred wording for Force and Sloan/Press Forward)

## Connect

Email, twitter, website, and Github icons/links would go here.

**Comment by rchampieux** [comment]
Hi there! Thanks @ryerbanta and @daniellecrobinson for the fantastic edits/additions.  I made a few updates (primarily, to "The Problem" section), but otherwise I think we're really, really close.  @saramannheimer and @swyoung can you take a final pass?

## The Problem

Great data related resources are out there. Many of you - data experts, practitioners, and learners alike -  share your data solutions, approaches, and ideas on the open web.  However,  finding these resources is hard, especially when materials with the potential for broad appeal are only shared with a specialized community.  Siloing data related resources leads to duplicate efforts and isolated discussions.  We want to break down disciplinary silos and create a space to share resources, amplify your work, and build an interdisciplinary community.

## Introducing Radian

Radian provides a community curated hub for sharing resources and discovering ideas about data science, visualization, data management, data policy, and applications for the public good. By highlighting and connecting the work you are using and creating in your practice we want to increase the discovery and recognition of data-related contributions on the open web, and foster a cross-disciplinary community built on information sharing.

## Contribute

Are you ready to amplify a great data-related resource? Use the PressForward bookmarklet to tell us about the awesome stuff you make and use in your work, and add some extra information to make is easier to discover and use by others. Our contributors will be recognized as Radian Editors-at-Large on our website and credited for the posts they contribute. Go here to learn more and get started (this would obviously point to more detailed info).

In addition to specific resource nominations, Radian uses the PressForward Plugin to aggregate and curate content from RSS feeds. If you have a feed, or know of a site Radian should follow, please submit the feed or site URL here. (again, will point somewhere).

## Team

Radian is a led by Robin Champieux and Sara Mannheimer, with additional help from Ryer Banta, Danielle Robinson, and Scott Young. We are generously funded by (add preferred wording for Force and Sloan/Press Forward)

## Connect

Email, twitter, website, and Github icons/links would go here.

**Comment by saramannheimer** [comment]
This looks so good! Thanks for all of your work, everyone. I just fixed a typo and suggested a deletion of one sentence. Looking forward to touching base tomorrow.

## The Problem

Great data related resources are out there. Many of you—data experts, practitioners, and learners alike—share your data solutions, approaches, and ideas on the open web.  However,  finding these resources is hard, especially when materials with the potential for broad appeal are only shared with a specialized community. --> I think we should remove this sentence, keep it super positive. :) ~~Siloing data related resources leads to duplicate efforts and isolated discussions.~~ <-- We want to break down disciplinary silos and create a space to share resources, amplify your work, and build an interdisciplinary community.

## Introducing Radian

Radian provides a community-curated hub for sharing resources and discovering ideas about data science, visualization, data management, data policy, and applications of data for the public good. By highlighting and connecting the work you are using and creating in your practice we want to increase the discovery and recognition of data-related contributions on the open web, and foster a cross-disciplinary community built on information sharing.

## Contribute

Are you ready to amplify a great data-related resource? Use the PressForward bookmarklet to tell us about the awesome stuff you make and use in your work, and add some extra information to make it easier to discover and use by others. Our contributors will be recognized as Radian Editors-at-Large and credited for the posts they contribute. Go here to learn more and get started (this would obviously point to more detailed info).

In addition to specific resource nominations, Radian uses the PressForward Plugin to aggregate and curate content from RSS feeds. If you have a feed, or know of a site Radian should follow, please submit the feed or site URL here. (again, will point somewhere).

## Team

Radian is a led by Robin Champieux and Sara Mannheimer, with additional help from Ryer Banta, Danielle Robinson, and Scott Young. We are generously funded by (add preferred wording for Force and Sloan/Press Forward)

## Connect

Email, twitter, website, and Github icons/links would go here.

**Comment by slwestcott** [comment]
HI, all (Robin asked me to take a look at the outreach language) - this looks really good. The only sentence I found it a little hard to parse was this one: "By highlighting and connecting the work you are using and creating in your practice we want to increase the discovery and recognition of data-related contributions on the open web, and foster a cross-disciplinary community built on information sharing." Might want to simplify a little. 

You also might want to plan on adding easy links to bookmarklet installation, etc, if you hadn't already. 

But that's about all I would suggest. I'm pretty excited to see how the project progresses!
